wxgui-src/wxmain.cc

```diff
@@ -235,8 +242,47 @@ void GameApp::new_game_core()
 }
 
 
-void GameApp::load_game(wxCommandEvent &)
-{
+void GameApp::save_game() {
+    if (this->autosave) {
+        do_error(this->filename);
+        FILE *fp = fopen(this->filename, "w");
+        if (fp) {
+            this->history.review(fp);
+            fclose(fp);
+            do_error("done.");
+        }
+    }
+}
+
+
+void GameApp::saveas_game(wxCommandEvent &) {
```

Comment:
This looks like it ought to be called `toggle_autosave`, and the GUI element ought to give some indication as to whether autosave is currently enabled or disabled.

There's definitely room for adding a proper "Save As", also, that would just save a transcript to the named file without enabling any kind of behind-the-scenes autosaving. This would allow the user to save his game, make some more moves, and if he started losing, he could backtrack by loading his save-game file.